include/veriblock/entities/altblock.hpp

```diff
@@ -123,13 +124,6 @@ inline void PrintTo(const AltBlock& block, ::std::ostream* os) {
 
 namespace std {
 
-template <>
-struct hash<std::vector<uint8_t>> {
-  size_t operator()(const std::vector<uint8_t>& x) const {
-    return std::hash<std::string>{}(std::string{x.begin(), x.end()});
-  }
-};
-
 template <>
 struct hash<altintegration::AltBlock> {
   size_t operator()(const altintegration::AltBlock& block) {
```

Comment:
No. Hashers of our entities can and should reside inside same hpp with an entity.

std::hash<std::vector<uint8_t>> is an exception, because std::vector is std type